electrum/lnsweep.py

```diff
@@ -328,35 +337,105 @@ def analyze_ctx(chan: 'Channel', ctx: Transaction):
     return ctn, their_pcp, is_revocation, per_commitment_secret
 
 
+def extract_funding_pubkeys_from_ctx(txin: TxInput) -> Optional[List[ecc.ECPubkey]]:
+    witness = txin.witness_elements()
+    if len(witness) == 4:
+        # we have OP_2(82) pk1 pk2 OP_2 OP_CHECKMULTISIG(174)
+        # TODO: could match_script_against_template
+        witness_script = witness[3]  # witness script is last element
+        script = [x for x in script_GetOp(witness_script)]
+        pubkey1 = script[1][1]
+        pubkey2 = script[2][1]
+        return [pubkey1, pubkey2]
+
+
+def create_sweeptx_their_backup_ctx(
+            *, chan: 'ChannelBackup',
+            ctx: Transaction,
+            sweep_address: str) -> Optional[Dict[str, SweepInfo]]:
+    txs = {}  # type: Dict[str, SweepInfo]
+    """Sweep to_remote, if we use anchors."""
```

Comment:
Maybe elaborate a bit on the scenario in the docstring.
e.g. 
> If we only have a backup, and the remote force-closed with their ctx, and anchors are enabled, we need to sweep to_remote.

Reply:
Done